_data/changelogs/component-combo-box.yml

```diff
@@ -2,7 +2,15 @@ title: Combo box
 type: component
 changelogURL:
 items:
-  - date: 2024-03-13
+  - date: NNNN-NN-NN
+    summary: Removed verbose aria instructions.
+    summaryAdditional: Combobox now only reads default screenreader instructions.
+    affectsAccessibility: true
+    affectsJavascript: true
+    githubPr: 6022
+    githubRepo: uswds
+    versionUswds: 3.5.0
+  - date: 2024-08-20
```

Comment:
**Issue (blocking):** Looks like the previous changelog date was accidentally updated. 
```suggestion
  - date: 2024-03-13
```